src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocLexer.g4

```diff
@@ -37,8 +37,7 @@ import java.util.*;
 
 }
 
-LEADING_ASTERISK : ( (' '|'\t') {_tokenStartCharPositionInLine == 0}? ) (' '|'\t')* '*'
-      | '*' {_tokenStartCharPositionInLine == 0}?
+LEADING_ASTERISK : (' '|'\t')* '*'
```

Comment:
> it wasn't recognizing leading asterisks after a newline 

You mean outside the `\r` new line change? If so, can you make a report to show this issue?